src/smartcontracts/typesystem/abiRegistry.ts

```diff
@@ -107,6 +112,12 @@ export class AbiRegistry {
         return <EnumType>result!;
     }
 
+    getExplicitEnum(name: string): ExplicitEnumType {
+        const result = this.customTypes.find((e) => e.getName() == name && e.hasExactClass(ExplicitEnumType.ClassName));
```

Comment:
Perhaps it does not help in this case, but I think it would be a good practice to use `===` wherever possible.